aws_securityhub/plugin.spec.yaml

```diff
@@ -707,9 +707,9 @@ types:
       required: false
   SHRecommendation:
     Url:
-      title: "Url"
+      title: "URL"
```

Comment:
This will be a major version bump since Title is used in the workflow language and workflows that use `{{[Url]}}` will now need to be updated to usae `{{[URL]}}`